src/main/java/tools/jackson/databind/JsonNode.java

```diff
@@ -983,6 +994,18 @@ public Iterator<JsonNode> elements() {
         return ClassUtil.emptyIterator();
     }
 
+    /**
+     * Method for accessing all value nodes of this Node, iff
+     * this node is a JSON Array or Object node. In case of Object node,
+     * field names (keys) are not included, only values.
+     * For other types of nodes, returns empty <code>Spliterator</code>.
+     *
+     * @since 3.0
+     */
+    public Spliterator<JsonNode> elementsSpliterator() {
```

Comment:
Based on `valueStream()` method, should probably be `value(s?)Spliterator()`

Reply:
+1 on `valueSpliterator`.